Readme.md

```diff
@@ -5,13 +5,15 @@
 [![Greenkeeper badge](https://badges.greenkeeper.io/Dean177/with-url-state.svg)](https://greenkeeper.io/)
 [![Npm](https://badge.fury.io/js/with-url-state.svg)](https://www.npmjs.com/package/with-url-state)
 
-Lifts state out of a react component and into the url
+This package allows applications to retrieve the state from a react component and appends it to the url.
```

Comment:
I like the expanded description, but can we keep the 'Lift state...' as a tagline, maybe having your new description just below the image.

Reply:
Would you be okay with "This package allows applications to lift the state out of a react component..." or would you rather have the 'Lifts state...' tagline as a standalone?

Reply:
Just as a standalone tagline please